lib/cocoapods/config.rb

```diff
@@ -314,6 +314,17 @@ def podfile_path_in_dir(dir)
       nil
     end
 
+    # Excludes the given dir from Time Machine backups.
+    #
+    # @param  [Pathname] dir
+    #         The directory to exclude from Time Machine backups.
+    #
+    # @return [void]
+    #
+    def exclude_from_backup(dir)
+      system('tmutil', 'addexclusion', dir.to_s, %i(out err) => File::NULL)
```

Comment:
should we check `Gem.win_platform?` before invoking this?

Reply:
We could also explicitly check for macOS but it might require pulling in [the OS gem](https://github.com/rdp/os) because [reliably detecting the operating system isn’t that easy](https://stackoverflow.com/q/170956). As we don’t care about the command’s exit status (and are basically just hoping for the best), invoking `tmutil` shouldn’t hurt. I agree, though, that it doesn’t make much sense to call it on any other operating system since `tmutil` definitely won’t be there.

Reply:
I dont mind personally if its best effort. We have used `Gem.win_platform?` in a couple of other spots like in `header_store.rb`.

Reply:
Done, I’ve added `return if Gem.win_platform?`.